### PR TITLE
Refs #13607 - Removed pulp.conf

### PR DESCRIPTION
**Pull request by parthaa**
The template is now removed and centralized in puppet-pulp

check https://github.com/Katello/puppet-pulp/pull/124

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -137,14 +137,6 @@
...
 
-    file {'/etc/httpd/conf.d/pulp.conf':
-      ensure  => file,
-      content => template('capsule/pulp.conf.erb'),
-      owner   => 'root',
-      group   => 'root',
-      mode    => '0644',
-    }
-
     file {'/etc/httpd/conf.d/pulp_nodes.conf':
```
Comment (ekohl) [inline manifests/init.pp]: What about this? Should this be moved as well?

Comment (ehelms) [inline manifests/init.pp]: Should line 138 also be removed now?

Comment (ekohl) [inline manifests/init.pp]: No, still used in httpd_pub.erb.

**Comment by ehelms** [comment]
ACK - thanks @parthaa !

**Pull request merged**